operator/src/main/resources/scripts/startServer.sh

```diff
@@ -221,7 +221,9 @@ function prepareMIIServer() {
         # expand the archive domain libraries to the domain lib
         cd ${DOMAIN_HOME}/lib || return 1
         ${JAVA_HOME}/bin/jar xf ${IMG_ARCHIVES_ROOTDIR}/${file} wlsdeploy/domainLibraries/
-
+        # Flatten the jars to the domain lib root
+        mv wlsdeploy/domainLibraries/* .
+        rm -fr wlsdeploy/
```

Comment:
>>  Should this code check to see if there are any unexpected files in the wlsdeploy dir once the mv is complete?

> I am not sure if this is necessary, the archive contains jar files in that location. If they put something wrong over there and if it causes problem starting the server then they should see it in the server log. Unless WDT has a command just to extract the libraries according to the model - it doesn't check much other than matching the path in the archive.

If the domain lib files aren't normally in an image archive, this will always generate an error message to stderr - which is confusing to see in a WL server pod log file.

If it's an error not to have domain lib files in an image archive, and the files are missing, then (a) the error will be a relatively hidden and cryptic 'wlsdeploy/domainLibraries/* not found' error on stderr in a WL pod log - not an ERROR trace message, and (b) since 'set -e' isn't on, the error will be ignored.  

So if  problem is possible but it's not always expected to have these files, then it'd be helpful to skip the flatten if there are no files in the directory.  Or if the the files must always be there, then the introspector job should verify that it exists and emit a helpful error before the WL pods start. or  to have the introspector check for this - it's harder to diagnose issues that are generated at the WL Server level...

Reply:
I agreed is not very common to use domain/lib,  at first I thought you are referring to checking what type of files are in the directory and whether they are valid jar files.  I have added check for directory existence before moving the files.

Reply:
Looks good.